### ISSUE TITLE
AssertionError: Image path does not exist: ./data/YoutubeVIS/valid_all_frames/v4/aed4e0b4c4/00000.jpg

### ISSUE THREAD

**Issue by 1seck** [haotian-liu/yolact_edge#200]
Hi there,

thank you for the repo :)
I cannot seem to find the download for the valid_all_frames/v4 subdir in YoutubeVIS. I did follow your install guide though. 

**Describe the bug**
I did all the installations.
I copied all the necessary files.
started: 
`python3 eval.py --trained_model=./weights/yolact_edge_vid_847_50000.pth --use_fp16_tensorrt`
I get the error:

> AssertionError: Image path does not exist: ./data/YoutubeVIS/valid_all_frames/v4/aed4e0b4c4/00000.jpg

**To Reproduce**
git clone yolact_edge
everything in [install](https://github.com/haotian-liu/yolact_edge#installation)
copied (hopefully) all data.

**Expected behavior**
python script should run without errors.

**Full logs**

>     evaluate(net, dataset)
>   File "eval.py", line 991, in evaluate
>     video_frames_data = next(data_loader_iter)
>   File "/home/ubuntu/miniconda3/envs/ye/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 363, in __next__
>     data = self._next_data()
>   File "/home/ubuntu/miniconda3/envs/ye/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 989, in _next_data
>     return self._process_data(data)
>   File "/home/ubuntu/miniconda3/envs/ye/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1014, in _process_data
>     data.reraise()
>   File "/home/ubuntu/miniconda3/envs/ye/lib/python3.7/site-packages/torch/_utils.py", line 395, in reraise
>     raise self.exc_type(msg)
> AssertionError: Caught AssertionError in DataLoader worker process 0.
> Original Traceback (most recent call last):
>   File "/home/ubuntu/miniconda3/envs/ye/lib/python3.7/site-packages/torch/utils/data/_utils/worker.py", line 185, in _worker_loop
>     data = fetcher.fetch(index)
>   File "/home/ubuntu/miniconda3/envs/ye/lib/python3.7/site-packages/torch/utils/data/_utils/fetch.py", line 44, in fetch
>     data = [self.dataset[idx] for idx in possibly_batched_index]
>   File "/home/ubuntu/miniconda3/envs/ye/lib/python3.7/site-packages/torch/utils/data/_utils/fetch.py", line 44, in <listcomp>
>     data = [self.dataset[idx] for idx in possibly_batched_index]
>   File "/home/ubuntu/y_edge/yolact_edge/data/youtube_vis.py", line 478, in __getitem__
>     return self.dataset.pull_video(video_idx, return_on_failure=True, full_video=True, max_images=self.max_images)
>   File "/home/ubuntu/y_edge/yolact_edge/data/youtube_vis.py", line 299, in pull_video
>     require_seeds=True)
>   File "/home/ubuntu/y_edge/yolact_edge/data/youtube_vis.py", line 341, in pull_frame
>     assert osp.exists(path), 'Image path does not exist: {}'.format(path)
> AssertionError: Image path does not exist: ./data/YoutubeVIS/valid_all_frames/v4/aed4e0b4c4/00000.jpg

**Environment:**
Ubuntu 18.04
Cuda 11.0
CuDNN 8.0.0

**Additional context**

**Comment by aneet-k** [comment]
Could you please tell how you got around this issue?

**Comment by 1tac11** [comment]
Hi there, some time ago, but first you have to install all the downloads from install readme (also the ones where the download is mostly not possible since the google drive quota is overdrawn), then copy them into the right subfolders. As I recall then the problem is that of the different frames subdirs zip-files you have to find the one with the right subdirs `aed4e0b4c4 ` etc. and then probably change the name of the files, I think it was from 0 5 10 15 to 0 1 2 3 etc.
KR

**Comment by aneet-k** [comment]
Thank you very much for your prompt reply. I literally needed somebody to help me with this right now! :)

I downloaded Youtube_VIS dataset from [here](https://youtube-vos.org/dataset/vis/), going to 2019 new version which leads[here](https://codalab.lisn.upsaclay.fr/competitions/6064#participate-get_data). Then I downloaded valid_all_frames (~5.8 GB) under the folder 'all_frames' (in google drive link).
But the folders, containing video frame images, in this data and the folder names referenced in annotation file for vis data provided by the authors (link in the install.md) are not overlapping. E.g. `aed4e0b4c4` is there in annotation-file (valid.v4.json) but not in the valid_all_frames dataset that I downloaded.

Maybe YOUTUBE_VIS updated their dataset. And the authors used some old data? 

p.s. As you said, there's mismatch in the names of the images also. 0 5 10 15..in the dataset and 1 2 3 .. in valid.v4.json
